jar-service/src/main/java/com/sixsq/slipstream/application/RootApplication.java

```diff
@@ -351,39 +349,24 @@ private void attachAction(RootRouter router) {
 	}
```

Comment:
Shouldn't attachAction() follow the introduced pattern and use guardAndAttach() as well?